live/app/router.js

```diff
@@ -1,7 +1,10 @@
-import Ember from 'ember';
+import EmberRouter from '@ember/routing/router';
+import { inject as service } from '@ember/service';
+import { run } from '@ember/runloop';
+import EmberObject from '@ember/object';
 import config from './config/environment';
```

Comment:
Yoh ! 
Pourquoi nommer la variable "EmberObject" ou "EmberRouter" au lieu de "object" et "router", j'ai l'impression qu'on les appelait comme ça depuis le début dans ce refacto

Reply:
C'est pas bloquant

Reply:
J'essaye de m'en tenir à des copier-coller de la doc, cf. https://guides.emberjs.com/v2.16.0/tutorial/routes-and-templates/#toc_an-about-route 

La l. 4 a d'ailleurs disparu.